pkg/apis/etcd/v1beta2/cluster.go

```diff
@@ -143,6 +143,14 @@ type PodPolicy struct {
 	// etcd cluster.
 	// The "etcd.version" annotation is reserved for the internal use of the etcd operator.
 	Annotations map[string]string `json:"annotations,omitempty"`
+
+	// busybox:latest uses uclibc which contains a bug that sometimes prevents name resolution
```

Comment:
Can you put these comment below on the BusyboxVersion comment? They are more related there.